### PR TITLE
Stricter linting

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: tests/test_auth.py [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-from flask_resty import (
-    Api, ApiError, AuthenticationBase, GenericModelView,
-    HasAnyCredentialsAuthorization, HasCredentialsAuthorizationBase,
-)
-from flask_resty.testing import assert_response
 from marshmallow import fields, Schema
 import pytest
 from sqlalchemy import Column, Integer, String
 
+from flask_resty import (
```
Comment (itajaja) [inline tests/test_auth.py]: don't like this much

**Comment by taion** [comment]
that's why it's marked WIP! i need to tweak the 4c one to not just die when running on py2

File: tests/test_auth.py (hunk 1 above)
Comment (itajaja) [inline tests/test_auth.py]: the

```py
)
from
```

is particularly ugly compared to `) from`. and also i'd be fine inlining many elements per row. anyhow, I don't really care, was just bitching :P

Comment (taion) [inline tests/test_auth.py]: the nice thing there is, when things break onto multiple lines, if you add something in the middle, you may need to re-wrap the line, which clutters up the diff

Comment (itajaja) [inline tests/test_auth.py]: ture

**Comment by taion** [comment]
Using 4C config now

File: .flake8 [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore = C813
```
Comment (itajaja) [inline .flake8]: why? for py2 compat?

Comment (taion) [inline .flake8]: yup; C813 is py3-only trailing commas

Comment (itajaja) [inline .flake8]: is it possible to install something like `flake8-config-4catalyzer[py2]` and conditionally trigger rules based on that?

Comment (itajaja) [inline .flake8]: I mean, we COULD drop py2 support on FR... :trollface:

Comment (taion) [inline .flake8]: I handled it upstream: https://github.com/4Catalyzer/python/pull/16

Comment (taion) [inline .flake8]: C813 is stuff in _source_; so things that violate C813 would be illegal syntax in py2

**Pull request merged**